architecture/core_objects/kubernetes_model.adoc

```diff
@@ -233,29 +233,56 @@ inside your application or container you simply bind to the local network on the
 port number for the service.
 
 Like pods, services are REST objects. To create a new service, one can be sent
-to the apiserver using a POST operation. The following example creates a new
-service with a name of "myapp", which resolves to TCP port 9376 on any pod with
-the "app=MyApp" label attached:
+to the apiserver using a POST operation. The following example shows the
+definition of a service with a name of `frontend`, which resolves to TCP port
+`8080` on any pod with the `name=frontend` label attached:
 
+.Service Definition
 ====
 
+[source,json]
 ----
 {
-  "id": "myapp",
-  "selector": {
-    "app": "MyApp"
-  },
-  "containerPort": 9376,
-  "protocol": "TCP",
-  "port": 8765
+    "kind": "Service",
+    "apiVersion": "v1beta3",
+    "metadata": {
+        "name": "frontend", <1>
+        "namespace": "myproject",
+        "selfLink": "/api/v1beta1/services/frontend?namespace=myproject",
+        "uid": "cb733818-ea98-11e4-a239-0800279696e1",
+        "resourceVersion": "204",
+        "creationTimestamp": "2015-04-24T15:44:29Z",
+        "labels": {
+            "template": "application-template-dockerbuild"
+        }
+    },
+    "spec": {
+        "ports": [
+            {
+                "name": "",
+                "protocol": "TCP",
+                "port": 5432,
+                "targetPort": 8080 <2>
+            }
+        ],
+        "selector": {
+            "name": "frontend" <3>
+        },
+        "portalIP": "172.30.217.106",
+        "sessionAffinity": "None"
+    },
+    "status": {}
 }
 ----
 
+<1> Service name of `frontend`.
+<2> Resolves to TCP port `8080`.
```

Comment:
Don't think "resolves" is the word we want here, that's more a DNS concept. I'd say "Forwards to container TCP port `8080`"